src/server/game/Scripting/ScriptDefines/PlayerScript.cpp

```diff
@@ -361,6 +361,14 @@ void ScriptMgr::OnPlayerLoadFromDB(Player* player)
     });
 }
 
+void ScriptMgr::OnPlayerBeforeLogout(Player* player)
```

Comment:
```suggestion
void ScriptMgr::OnBeforePlayerLogout(Player* player)
```

My bad! Doing several things at the same time.

Reply:
No biggie.  It's an easy change to make.  Thanks for reviewing.